Sources/MongoSwift/Collection.swift

```diff
@@ -1,5 +1,23 @@
 import libmongoc
 
+/// An index to "hint" or force MongoDB to use when performing a query.
+public enum Hint: Encodable {
```

Comment:
open to any better naming suggestions for these cases

Reply:
The [`find` docs](https://docs.mongodb.com/manual/reference/command/find/) refer to both forms as an "index specification", although [`cursor.hint()`](https://docs.mongodb.com/manual/reference/method/cursor.hint/) refers only to the document as an "index specification". The only alternative I might consider is "indexKeys", but I'm comfortable with using "indexSpec" as it should be quite clear to users.